### DISCUSSION TITLE
Welcome to CloneBot V2 Discussions!

### DISCUSSION THREAD

**Discussion by TheCaduceus** [TheCaduceus/CloneBot_V2#25] [Ideas]
## 👋 Welcome!
  CloneBot V2 Discussions enables users to interact and help each other. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by Coss27** [comment]
I deploy bot V2 on my PC,The bot work good when I move files between my team drives, but when I put a Google drive link, I get this error:

Unable to fetch Google drive link
